modules/localgov_services_page/config/install/core.entity_form_display.node.localgov_services_page.default.yml

```diff
@@ -2,15 +2,16 @@ langcode: en
 status: true
 dependencies:
   config:
+    - entity_browser.browser.page_components
```

Comment:
I guess this is a paragraphs requirement that isn't strictly related to this commit?